src/constants/europe.ts

```diff
@@ -106,7 +106,7 @@ const getKiaEnvironment = ({
   const host = 'prd.eu-ccapi.kia.com:8080';
   const baseUrl = `https://${host}`;
   const clientId = 'fdc85c00-0a2f-4c64-bcb4-2cfb1500730a';
-  const appId = 'e7bcd186-a5fd-410d-92cb-6876a42288bd';
+  const appId = 'a2b8469b-30a3-4361-8e13-6fceea8fbe74';
```

Comment:
Is this change the only thing needed or is there also some stamp changes?

Reply:
Also stamp changes, I changed those in the other repo (https://github.com/neoPix/bluelinky-stamps/pull/7)

Reply:
@PierreLevres tested it with the new appId and the raw stamps file from your linked PR, but the problem is not solved :(

```
 debug: Bluelinky is logging in automatically, to disable use autoLogin: false
 debug: @EuropeController.login: Trying to sign in with EuropeanBrandAuthStrategy
 debug: @EuropeController.login: Authenticated properly with user and password

/home/nesta/workspace/daham/bluelinky/src/tools/common.tools.ts:19
    return new ManagedBluelinkyError(
           ^
ManagedBluelinkyError: @EuropeController.login: 
   [400] Bad Request on
    [POST] https://prd.eu-ccapi.kia.com:8080/api/v1/spa/notifications/register - 
    {"retCode":"F","resCode":"4002","resMsg":"Invalid request body - Invalid parameter.","msgId":"c432c802-4932-4e9d-8f88-3d1c533a7c9d"}
    at Object.exports.manageBluelinkyError (/home/nesta/workspace/daham/bluelinky/src/tools/common.tools.ts:19:12)
    at EuropeanController.<anonymous> (/home/nesta/workspace/daham/bluelinky/src/controllers/european.controller.ts:257:13)

```

+ It looks like the problem affects KIA and Hyundai.